workflow/tests/test_workflow.py

```diff
@@ -1,50 +1,38 @@
-import os 
+import os
 import subprocess
 
-
-class TestWorkflow:
+CONFIGS = {
+    'default':'default.yaml',
+    'tps-cytoscape':'tps-cytoscape.yaml',
+    'tps-no-container':'tps-no-container.yaml'
+}
+class TestWorkFlow:
 
     @classmethod
     def setup_method(self):
         self.tps_root = os.getcwd()
-        self.config = os.path.join(
-            self.tps_root, 'workflow', 'tests', 'test_standards', 'test_default_config.yaml'
-        )
-        self.tps_runner = os.path.join(self.tps_root, 'tps_runner.py')
-
-    def test_tps(self):
-        print(f'tps_root: {self.tps_root}')
-        print(f'tps_runner: {self.tps_runner}')
-        assert os.path.isfile(self.tps_runner)
-        assert os.path.isdir(self.tps_root)
-        call = ['python', self.tps_runner, 
-                '--config', self.config, '--execute', 'tps']
+        self.tps_main = os.path.join(self.tps_root, 'tps_runner.py')
+    
+    def test_workflow_default(self):
+        config = os.path.join(self.tps_root,'workflow' ,'tests', 'standards', CONFIGS['default'])
+        call = ['python', self.tps_main, 
+                '--config', config, '--execute', 'tps']
         assert subprocess.run(call, check=True).returncode == 0
 
-        outputs = [
```

Comment:
We discussed restoring this type of test that confirms the expected output files exist after the workflow runs.